glean-core/python/glean/glean.py

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from . import _ffi
+
+
+class Glean:
+    """
+    The main Glean API.
+
+    This is exposed through the global `Glean` object.
```

Comment:
Can you copy over the Glean object description from either the Kotlin or Swift code?

Reply:
[That's where I got this from](https://github.com/mozilla/glean/blob/a1164301b9769d54354cc214d5dd5f56b56e87b9/glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt#L46)

Reply:
Oh, but I see there's a docstring on the variable at the bottom.  Python can't do that, but we can put that content here.

Reply:
> 
> 
> Oh, but I see there's a docstring on the variable at the bottom. Python can't do that, but we can put that content here.

Right! I was looking at the one on the bottom, good call on merging them.